<!DOCTYPE html>
<html >
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Byron J. Smith">
  <title>Byron J. Smith</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-40659359-4', 'auto');
  ga('send', 'pageview');

</script>
</head>
<body>
<header>
<h1 class="title">Byron J. Smith</h1>
<div class="contact">
  /
  <a id="email-link" href="mailto:me@byronjsmith.com">me@byronjsmith.com</a> / 
  <a id="twitter-link" href="https://twitter.com/ByronJSmith">@ByronJSmith</a> / 
  <a id="linkedin-link" href="https://www.linkedin.com/in/bsmith89">LinkedIn</a> / 
  <a id="orcid-link" href="https://orcid.org/0000-0002-0182-404X">ORCiD</a> / 
  <a id="github-link" href="https://github.com/bsmith89">Github</a> / 
  <a id="website-link" href="http://blog.byronjsmith.com">Website</a> / 
</div>
</header>
<hr>
<h2 id="research-focus">Research Focus</h2>
<ul>
<li>I develop and use bioinformatic tools and statistical models to integrate diverse data towards understanding complex microbial communities.</li>
</ul>
<h2 id="publications">Publications</h2>
<ul>
<li><p>[in preperation] <strong>B.J. Smith</strong>, T.M. Schmidt. <em>Muribaculaceae</em> genomes assembled from metagenomes suggest genetic drivers of differential response to acarbose treatment in mice.</p></li>
<li><p>[2018 preprint] <strong>B.J. Smith</strong>, R.A. Miller, A.A. Ericsson, R. Strong, D.E. Harrison, T.M. Schmidt. Changes in the gut microbiota and fermentation products associated with enhanced longevity in acarbose-treated mice. <em>bioRxiv</em>. doi:<a href="https://doi.org/10.1101/311456">10.1101/311456</a> (Submitted to <em>BMC Microbiology</em>)</p></li>
<li><p>[2015] S.F. Stoddard, <strong>B.J. Smith</strong>, R. Hein, B.R.K. Roller, T.M. Schmidt. rrnDB: Improved tools for interpreting rRNA gene abundance in Bacteria and Archaea and a new foundation for future development. <em>Nucleic Acids Research</em>. doi:<a href="https://doi.org/10.1093/nar/gku1201">10.1093/nar/gku1201</a></p></li>
</ul>
<h2 id="posters-and-presentations">Posters and Presentations</h2>
<ul>
<li>“Changes in the gut microbiota and fermentation products associated with enhanced longevity in acarbose-treated mice.” Presented at:
<ul>
<li>[2018] (invited talk) <strong>Pollard Lab Meeting</strong>, University of California, San Francisco, invited by Dr. Katherine Pollard</li>
<li>[2018] (<a href="http://byronjsmith.com/keystone18X3_poster.pdf">poster</a>) <strong>Keystone Symposium on Manipulation of the Gut Microbiota for Metabolic Health</strong>, Banff, Alberta</li>
<li>[2017] (invited talk) <strong>Lynch Lab Meeting</strong>, University of California, San Francisco, invited by Dr. Susan Lynch</li>
<li>[2017] (talk) <strong>Microbiome Group Seminar Series</strong>, University of Michigan</li>
<li>[2017] (poster) <strong>American Society for Microbiology - Microbe</strong>, New Orleans, Louisiana</li>
</ul></li>
<li><p>[2017] “Automating reproducible data analysis pipelines with Snakemake” (<a href="http://blog.byronjsmith.com/snakemake-analysis.html">invited talk</a>), <strong>Department of Research Computing - Joint User Groups Meeting</strong>, Boise State University</p></li>
<li><p>[2016] “The longevity enhancing drug Acarbose modulates the gut microbiome and its fermentation products” (poster), <strong>Michigan Meeting on the Microbiome</strong>, University of Michigan</p></li>
<li><p>[2014] “Slow but successful: spatial structure and the evolution of efficiency” (talk), <strong>Pre-Candidacy Seminar</strong>, Department of Ecology and Evolutionary Biology, University of Michigan</p></li>
<li><p>[2011] “GABA signaling and reception in the developing nervous system” (talk) <strong>Spring Honors Colloquium</strong>, The College of William &amp; Mary</p></li>
<li><p>[2010] “The role of calcium in neural phenotype differentiation in developing Xenopus” (poster), <strong>Undergraduate Research Symposium</strong>, The College of William &amp; Mary</p></li>
</ul>
<h2 id="grants-and-fellowships">Grants and Fellowships</h2>
<ul>
<li>[2016-2018] Integrated Training in Microbial Systems (ITiMS) Fellowship
<ul>
<li>Co-advised by Dr. Thomas M. Schmidt and Dr. Aaron A. King</li>
</ul></li>
<li>[2015-2016] Instrumental in preparing a UM Glenn Center for Aging Research grant which was funded for $50,000
<ul>
<li>Principal Investigator: Dr. Thomas M. Schmidt</li>
</ul></li>
<li><p>[2013-2014] UM Ecology &amp; Evolutionary Biology Departmental Block Grant</p></li>
<li><p>[2011-2012] Excellence in BioMolecular Science Fellowship from the College of Natural Science, Michigan State University</p></li>
<li><p>[2008-2010] Howard Hughes Medical Institute Student Research Awards</p></li>
<li><p>[2008] Howard Hughes Medical Institute Freshman Research Grant</p></li>
</ul>
<h2 id="education">Education</h2>
<ul>
<li>[2013-2018] <strong>University of Michigan</strong>, Ann Arbor, MI
<ul>
<li>Ph.D. in Ecology and Evolutionary Biology</li>
<li><a href="http://byronjsmith.com/dissertation.pdf">Dissertation</a>: “Integrated Analysis of the Gut Microbiota and Their Fermentation Products in Mice Treated with the Longevity Enhancing Drug Acarbose”</li>
<li>Embedded Masters in Ecology and Evolutionary Biology, granted in 2016</li>
<li>GPA: 4.0</li>
<li>Research Advisor: Dr. Thomas M. Schmidt</li>
</ul></li>
<li>[2011-2013] <strong>Michigan State University</strong>, East Lansing, MI
<ul>
<li>Graduate student, Microbiology and Molecular Genetics</li>
<li>GPA: 4.0</li>
<li>Affiliate, BEACON Center for The Study of Evolution in Action</li>
<li>Affiliate, Great Lakes Bioenergy Research Center</li>
<li>Research Advisor: Dr. Thomas M. Schmidt</li>
</ul></li>
<li>[2007-2011] <strong>The College of William &amp; Mary</strong>, Williamsburg, VA
<ul>
<li>B.S. Biology</li>
<li>Graduated <em>Summa Cum Laude</em></li>
<li>GPA: 3.80 (overall) / 4.00 (major)</li>
<li>Minor: Mathematics</li>
<li>Research Advisor: Dr. Margarete S. Saha</li>
</ul></li>
</ul>
<h3 id="select-coursework-and-workshops">Select Coursework and Workshops</h3>
<ul>
<li><p>[2015] <strong>Train the Trainers: Teaching Adults Researchers to Program</strong>, Michigan State University</p></li>
<li><p>[2014] <strong>Guarda Workshop in Evolutionary Biology</strong>, University of Basel and ETH Zürich, Switzerland</p></li>
<li><p>[2012] <strong>Software Carpentry Bootcamp</strong>, Michigan State University</p></li>
<li><p>[2012] <strong>Enhancing Linkages Between Mathematics and Ecology</strong>, Kellogg Biological Station, Hickory Corners, MI</p></li>
<li><p>[2011] <strong>Microbial Metagenomics Workshop</strong>, Michigan State University</p></li>
</ul>
<h2 id="honors-and-awards">Honors and Awards</h2>
<ul>
<li><p>[2011] Nominated for Michigan State University Distinguished Fellowship or University Enrichment Fellowship</p></li>
<li><p>[2007, 2008, 2009, 2011] The College of William &amp; Mary Dean’s List for Academic Excellence</p></li>
<li><p>[2008] Inducted into Alpha Lambda Delta and Phi Eta Sigma Academic Honor Societies</p></li>
</ul>
<h2 id="research-training">Research Training</h2>
<ul>
<li>Advisor: Dr. Katherine S. Pollard
<ul>
<li>[2019-Present] University of California, San Fracisco</li>
</ul></li>
<li>Advisor: Dr. Thomas M. Schmidt
<ul>
<li>[2012-2018] Michigan State University and University of Michigan</li>
</ul></li>
<li>Advisor: Dr. Margaret S. Saha
<ul>
<li>[2008-2011] The College of William &amp; Mary</li>
</ul></li>
<li>Advisor: Dr. Mary A. Voytek
<ul>
<li>[2006-2007] The United States Geological Survey</li>
</ul></li>
</ul>
<h2 id="public-datasets">Public Datasets</h2>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA448009">PRJNA448009</a>: 16S rRNA gene libraries of fecal samples from acarbose treated and control mice
<ul>
<li>Associated metabolite profiles and longevity data also available</li>
</ul></li>
</ul>
<h2 id="teaching">Teaching</h2>
<ul>
<li>[2015, 2018] Graduate Student Instructor, <strong>Introductory Biology Lab: The Human Microbiome</strong>, University of Michigan
<ul>
<li>Assisted during three semesters in designing and running laboratory activities and lectures for a novel, research based, introductory biology lab</li>
<li>Course Instructors: Dr. Thomas M. Schmidt, Dr. Kristi MacCready, and Dr. Arvind Venkateraman</li>
</ul></li>
<li>[2015-present] Instructor, <strong>Software Carpentry Bootcamps</strong>,
<ul>
<li>Two day workshops in collaboration with other instructors on computing skills for researchers: programming, the command-line version control, data management, etc.</li>
<li>Students are early career scientists (primarily postdocs and graduate students)</li>
<li>Workshops at Stanford University, Harvard Medical School, University of Pennsylvania, University of California, San Francisco, University of New Hampshire, University of Michigan, Wayne State University, Lawrence Berkeley National Laboratory</li>
</ul></li>
<li>[2017] Teaching Assistant, <strong>Data Intensive Biology Summer Research Institute</strong> hosted by the University of California, Davis
<ul>
<li>Assisted with week-long workshops on “Environmental Metagenomics” and “Non-Model Organism RNA-Seq”</li>
<li>Organizer: Dr. C. Titus Brown</li>
</ul></li>
<li>[2016] Instructor, <strong>Bodega Marine Laboratory Bioinformatics Workshop</strong>, hosted by the University of California, Davis
<ul>
<li>Invited to teach a half-day lesson titled <a href="http://blog.byronjsmith.com/make-analysis.html"><em>GNU Make for Reproducible Analysis Pipelines</em></a></li>
<li>Workshop Organizer: Dr. C. Titus Brown</li>
</ul></li>
<li>[2014] Teaching Assistant, <strong>Software Carpentry Bootcamp</strong>, University of Michigan
<ul>
<li>Assisted instructors teaching basic computing skills to early career scientists</li>
</ul></li>
<li>[2013] Graduate Student Instructor, <strong>General Ecology</strong>, University of Michigan
<ul>
<li>Course Instructor: Dr. Aaron A. King</li>
</ul></li>
<li>[2012] Teaching Assistant, <strong>Microbial Genomics Lab</strong>, Michigan State University
<ul>
<li>Course Instructor: Dr. Robert A. Britton</li>
</ul></li>
</ul>
<h2 id="service-and-outreach">Service and Outreach</h2>
<ul>
<li><p>[2018-present] <strong>Software Carpentry Curriculum Advisory Committee</strong> member</p></li>
<li>[2016-2018] <strong>University of Michigan Software Carpentry</strong> group member
<ul>
<li>Actively involved in developing a community of Software and Data Carpentry instructors, learners, and advocates</li>
</ul></li>
<li>[2017] <strong>Models: at the intersection of data and discovery.</strong>
<ul>
<li>Collaborated in proposing a meeting for the directors and students of the Burroughs Wellcome Fund’s Institutional Program Unifying Population and Laboratory Based Sciences</li>
<li>Held at the University of Michigan</li>
<li>$150,000 budget</li>
</ul></li>
<li>[2016-2017] <strong>Ecology and Evolutionary Biology Statistics Office Hours</strong> organizer
<ul>
<li>Co-founded and organized monthly statistics help sessions for graduate students</li>
</ul></li>
<li><p>[2016] Graduate Student Representative to the Department of Ecology and Evolutionary Biology <strong>Executive Committee</strong> during the fall term</p></li>
<li><p>[2014-2015] Graduate Student Representative to the Department of Ecology and Evolutionary Biology <strong>Seminar Committee</strong></p></li>
<li><p>[2015] National Ocean Sciences Bowl, “<strong>Great Lakes Bowl</strong>” volunteer</p></li>
<li><p>[2013-2014] Michigan <strong>Xplore Engineering</strong> “Microbiology and Environmental Engineering” workshop team</p></li>
</ul>
<div class="permalink">
  <a href="http://www.byronjsmith.com/resume.html">
      Version: 2019-01-11
    </a>
</div>
</body>
</html>
