---
name: Byron J. Smith, PhD

email: me@byronjsmith.com
twitter: ByronJSmith
linkedin: bsmith89
github: bsmith89
orcid: 0000-0002-0182-404X
gscholar: ZtIlAngAAAAJ
website: http://blog.byronjsmith.com
header-includes: >
    \usepackage{fullpage}
    \usepackage{fontspec}
    \setmainfont{Verdana}

analytics: UA-40659359-4

version: 2023-11-02
current: http://www.byronjsmith.com/resume.html
...


## Research Focus ##

I develop widely applicable bioinformatic and statistical methods to understand
the causes and consequences of enormous, intraspecific, genomic diversity among
microbes residing in the human gut.


## Publications ##

-   [in prep] **B.J. Smith**, X. Jin, C. Zhao, K.S. Pollard.
    StrainPGC: Accurate inference of intraspecific microbial gene content
    variation in large collections of shotgun metagenomic data.
    (for submission to _Genome Research_)

-   [submitted] B. Zhang, K. Magnaye, E. Stryker, J. Moltzau Anderson, C. E.
    Porsche, S. Hertz, K. E. McCauley, **B. J. Smith**, M. Zydek, K. S.
    Pollard, A. Ma, N. El-Nachef, S. V. Lynch. Sustained mucosal colonization
    and fecal metabolic dysfunction by Bacteroides associates with fecal
    microbial transplant failure in ulcerative colitis patients. Submitted to
    _Scientific Reports_.

-   [2022] C. Zhao, M. Goldman, **B.J. Smith**, K.S. Pollard. Genotyping
    microbial communities with MIDAS2: from metagenomic reads to allele
    frequency tables. _Current Protocols in Bioinformatics_
    doi:[10.1002/cpz1.604](https://doi.org/10.1002/cpz1.604)

-   [2022]
    **B.J. Smith**, X. Li, Z.J. Shi, A. Abate, K.S. Pollard. Scalable microbial
    strain inference in metagenomic data using StrainFacts.
    _Frontiers in Bioinformatics_
    doi:[10.3389/fbinf.2022.867386](https://doi.org/10.3389/fbinf.2022.867386)

-   [2022]
    **B.J. Smith**, Y.M. Piceno, M. Zydek, B. Zhang, L. Aboud Syriani,
    J.P. Terdiman, Z. Kassam, A. Ma, S.V. Lynch, K.S. Pollard, N. El-Nachef.
    Strain-resolved analysis in a randomized trial of antibiotic pretreatment
    and maintenance dose delivery mode with fecal microbiota transplant for
    ulcerative colitis. _Scientific Reports_
    doi:[10.1038/s41598-022-09307-5](https://doi.org/10.1038/s41598-022-09307-5)

-   [2021]
    **B.J. Smith**, R.A. Miller, T.M. Schmidt. _Muribaculaceae_ genomes assembled from
    metagenomes suggest genetic drivers of differential response to acarbose
    treatment in mice. _mSphere_
    doi:[10.1128/msphere.00851-21](https://doi.org/10.1128/msphere.00851-21)

-   [2021] **B.J. Smith**, K.S. Pollard. Evaluation of Le et al.: Challenges
    and opportunities for using data to understand equitability in science.
    _Cell Systems_
    doi:[10.1016/j.cels.2021.08.005](https://doi.org/10.1016/j.cels.2021.08.005)

-   [2020]
    N. El-Nachef, **B.J. Smith**, Y. Piceno, M. Zydek, L.
    Syriani, J.P. Terdiman, A. Ma, Z. Kassam, K. Pollard, S. Lynch.
    Antibiotic pretreatment prior to fecal microbiota transplantation increases
    rates of remission in patients with mild to moderate ulcerative colitis:
    Results from a pilot randomized clinical trial.
    _Gastroenterology_ (Digestive Disease Week Abstract)
    doi:[10.1016/S0016-5085(20)30716-2](https://doi.org/10.1016/S0016-5085(20)30716-2)

-   [2019]
    **B.J. Smith**, R.A. Miller, A.A. Ericsson, R. Strong, D.E. Harrison,
    T.M. Schmidt. Changes in the gut microbiome and fermentation products
    concurrent with enhanced longevity in acarbose-treated mice.
    _BMC Microbiology_.
    doi:[10.1186/s12866-019-1494-7](https://doi.org/10.1186/s12866-019-1494-7)

-   [2015]
    S.F. Stoddard, **B.J. Smith**, R. Hein, B.R.K. Roller, T.M. Schmidt.
    rrnDB: Improved tools for interpreting rRNA gene abundance in Bacteria and
    Archaea and a new foundation for future development.
    _Nucleic Acids Research_.
    doi:[10.1093/nar/gku1201](https://doi.org/10.1093/nar/gku1201)


## Posters and Presentations ##

-   [2023] "Phylogenetic diversity patterns among gastrointestinal bacterial strains"
    (poster) **Genome Informatics**, Cold Spring Harbor Lab
    - Presented by Veronika Dubinkina

-   "Resolving strain-level gene content variation from large, metagenomic datasets"
    ([poster](https://byronjsmith.com/microbialpop2023_poster.pdf) / [abstract](https://byronjsmith.com/grc_micropop_2023_abstract.pdf))
     - [2023] **Microbial Population Biology Gordon Research Conference**, Andover, NH
     - [2023] **Joint Berkeley Initiative for Microbiome Sciences: Microbiomes in a Changing Planet**, Berkeley, CA

-   [2023]
    "Strain-resolved inference of microbial gene content in fecal microbiota
    transplantation to treat ulcerative colitis."
    ([poster](https://byronjsmith.com/rainin2023_poster.pdf) / [abstract](https://byronjsmith.com/rainin2023_abstract.pdf))
    **Kenneth Rainin Foundation Innovations Symposium**, San Francisco, CA

-   [2022]
    "Strain tracking and reconstruction in large metagenomic datasets."
    ([invited talk](https://byronjsmith.com/qbc_retreat_2022_slides.pdf))
    **Quantitative Biosciences Consortium Retreat**, Santa Cruz, CA

-   [2022] "Strain-resolved bacterial genome reconstruction in large,
    metagenomic datasets"
    ([poster](https://byronjsmith.com/noyce_symposium_2022_poster.pdf))
    **University of California Noyce Initiative Symposium**, Santa Barbara, CA

-   [2022]
    "Strain-resolved inference of microbial gene content in large metagenomic datasets."
    ([talk](https://byronjsmith.com/cshl_microbiome_2022_slides.pdf) /
    [abstract](https://byronjsmith.com/cshl_microbiome_2022_abstract.pdf))
    **CSHL - Microbiome Meeting**, Cold Spring Harbor, NY

-   [2022]
    "Scaling microbial strain inference to tens of thousands of metagenomes with fuzzy genotypes."
    ([invited talk](https://byronjsmith.com/imsi2022_slides_with_notes.pdf))
    Workshop on **Multiscale Microbial Communities: Dynamical Models, Ecology, and One Health**,
    Institute for Mathematical and Statistical Innovation, Chicago, IL

-   [2021] "Bacterial genotype deconvolution in shotgun
    metagenomic reads using fuzzy genotypes"
    ([poster](https://byronjsmith.com/probgen2021_poster.pdf))
    **Probabilistic Modeling in Genomics**, Cold Spring Harbor Lab

-   [2020] "Identifying and tracking bacterial strains in metagenomic libraries"
    ([talk](https://byronjsmith.com/toolstech2020.pdf))
    **Tools & Tech RIPS**, The Gladstone Institutes, San Francisco, California

-   [2020] "Strain-level bacterial reconstruction and inference in patients
    receiving fecal microbiota transplantation for ulcerative colitis"
    ([talk](https://byronjsmith.com/ignite2020_slides.pdf))
    **Microbiome Ignite Series**, University of California, San Francisco

-   [2019] "Improved methods for strain-specific genome reconstruction"
    ([lightning talk](https://byronjsmith.com/biohubconfab2019_slides.pdf))
    **Chan Zuckerberg Biohub Inter-lab Confab**,
    San Francisco, California

-   [2019] "_Muribaculaceae_ Genomes assembled from metagenomes suggest genetic
    drivers of differential response to acarbose treatment in mice"
    ([poster](https://byronjsmith.com/asm2019_poster.pdf)),
    **American Society for Microbiology - Microbe**, San Francisco, California

-   "Changes in the gut microbiota and fermentation products associated
    with enhanced longevity in acarbose-treated mice."
    Presented at:
    -   [2018] \(invited talk\)
        **Pollard Lab Meeting**, University of California,
        San Francisco, invited by Dr. Katherine S. Pollard
    -   [2018] \([poster](https://byronjsmith.com/keystone18X3_poster.pdf)\)
        **Keystone Symposium on Manipulation of the Gut
        Microbiota for Metabolic Health**, Banff, Alberta
    -   [2017] \(invited talk\) **Lynch Lab Meeting**,
        University of California, San Francisco, invited by Dr. Susan V. Lynch
    -   [2017] \(talk\) **Microbiome Group Seminar Series**, University of
        Michigan
    -   [2017] \(poster\) **American Society for Microbiology - Microbe**,
        New Orleans, Louisiana

-   [2017] "Automating reproducible data analysis pipelines with Snakemake"
    ([invited talk](https://blog.byronjsmith.com/snakemake-analysis.html)),
    **Department of Research Computing - Joint User Groups Meeting**,
    Boise State University

-   [2016] "The longevity enhancing drug Acarbose modulates
    the gut microbiome and its fermentation products" (poster),
    **Michigan Meeting on the Microbiome**, University of Michigan

-   [2014] "Slow but successful: spatial structure and the evolution of
    efficiency" (talk), **Pre-Candidacy Seminar**, Department of Ecology and
    Evolutionary Biology, University of Michigan

-   [2011] "GABA signaling and reception in the developing nervous system"
    (talk) **Spring Honors Colloquium**, The College of William \& Mary

-   [2010] "The role of calcium in neural phenotype differentiation in
    developing Xenopus" (poster), **Undergraduate Research Symposium**,
    The College of William \& Mary


## Grants and Fellowships ##

-   [2022-2023] Contributed to the writing of a Helmsley Charitable Trust grant
    that was funded for \$245,905. Principal Investigators: Drs. Dirk Haller,
    Tobias Schwerd, Katherine Pollard

-   [2021-2022] Computational Innovation Postdoctoral Fellowship from the
    Noyce Initiative for Digital Transformation in Computational Biology and
    Health Data Science for my proposal, "Strain-resolved microbiome-wide
    association studies for the discovery of hidden, microbial drivers of
    disease", funded for \$75,000

-   [2016-2018] Integrated Training in Microbial Systems (ITiMS) Graduate
    Fellowship co-advised by Dr. Thomas M. Schmidt and Dr. Aaron A. King

-   [2015-2016] Instrumental in preparing a UM Glenn Center for Aging Research
    grant that was funded for \$50,000.
    Principal Investigator: Dr. Thomas M. Schmidt

-   [2013-2014] UM Ecology & Evolutionary Biology Departmental Block Grant

-   [2011-2012] Excellence in BioMolecular Science Fellowship from the College
    of Natural Science, Michigan State University

-   [2008-2010] Howard Hughes Medical Institute Student Research Awards

-   [2008] Howard Hughes Medical Institute Freshman Research Grant


## Honors and Awards ##

-   [2023] Distinguished Achievement in Scientific Leadership Award from the
    Gladstone Institute of Data Science and Biotechnology

    -   This award is granted to exceptionally collaborative scientists who
        take on leadership roles at Gladstone and demonstrate exemplary
        collaboration, mentorship, and integrity that reaches beyond their own
        lab

-   [2011] Nominated for Michigan State University Distinguished Fellowship or
    University Enrichment Fellowship

-   [2007, 2008, 2009, 2011] The College of William \& Mary
    Dean's List for Academic Excellence

-   [2008] Inducted into Alpha Lambda Delta and Phi Eta Sigma Academic Honor
    Societies


## Education and Professional Experience ##

-   [2019-present] Postdoctoral Scholar
    -   **Gladstone Institute** for Data Science and Biotechnology, San Francisco, CA
    -   Research Advisor: Dr. Katherine S. Pollard

-   [2013-2018] PhD in Ecology and Evolutionary Biology
    -   **University of Michigan**, Ann Arbor, MI
    -   [Dissertation](https://byronjsmith.com/dissertation.pdf): "Integrated
        Analysis of the Gut Microbiota and Their Fermentation Products in Mice
        Treated with the Longevity Enhancing Drug Acarbose"
    -   Embedded Masters in Ecology and Evolutionary Biology, granted in 2016
    -   GPA: 4.0
    -   Research Advisor: Dr. Thomas M. Schmidt

-   [2011-2013] Graduate Student in Microbiology and Molecular Genetics
    -   **Michigan State University**, East Lansing, MI
    -   GPA: 4.0
    -   Affiliate, BEACON Center for The Study of Evolution in Action
    -   Affiliate, Great Lakes Bioenergy Research Center
    -   Research Advisor: Dr. Thomas M. Schmidt

-   [2007-2011] Bachelor of Science in Biology
    -   **The College of William \& Mary**, Williamsburg, VA
    -   Graduated _Summa Cum Laude_
    -   GPA: 3.80 (overall) / 4.00 (major)
    -   Minor: Mathematics
    -   Research Advisor: Dr. Margaret S. Saha


## Research Training ##

-   [2019-Present] Advised by **Dr. Katherine S. Pollard**
    at the University of California, San Francisco

-   [2012-2018] Advised by **Dr. Thomas M. Schmidt**
    at Michigan State University and University of Michigan

-   [2008-2011] Advised by **Dr. Margaret S. Saha**
    at The College of William \& Mary

-   [2006-2007] Advised by **Dr. Mary A. Voytek**
    at the United States Geological Survey


### Select Coursework and Workshops ###

-   [2023]
    **Scientific Leadership and Management**,
    The Gladstone Institutes, San Francisco, CA
    -  Three-day course designed for senior postdocs and junior faculty to
       learn about "leadership styles, people management, inclusive leadership,
       and conflict resolution".

-   [2015]
    **Train the Trainers: Teaching Adult Researchers to Program**,
    Michigan State University

-   [2014] **Guarda Workshop in Evolutionary Biology**, University of Basel and
    ETH ZÃ¼rich, Switzerland

-   [2012] **Software Carpentry Bootcamp**, Michigan State University

-   [2012] **Enhancing Linkages Between Mathematics and Ecology**,
    Kellogg Biological Station, Hickory Corners, MI

-   [2011] **Microbial Metagenomics Workshop**, Michigan State University


## Public Datasets and Software ##

-   [StrainFacts](https://github.com/bsmith89/StrainFacts):
    Computational tool to identify and track microbial strains across large
    numbers of metagenomic samples.

-   [PRJNA737472](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA737472):
    Longitudinal fecal metagenomes and 16S rRNA gene libraries from ulcerative colitis
    patients receiving fecal microbiota transplants with and without antibiotic
    pretreatment and with maintenance dosing via either capsules or enema.
    -   Associated bile acid profiles are also available

-   [PRJNA448009](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA448009):
    Paired metagenomes and 16S rRNA gene libraries of fecal samples from
    acarbose treated and control mice, as well as carefully curated
    metagenome assembled genomes from nine members of the understudied
    bacterial family, the _Muribaculaceae_
    -   Associated metabolite profiles and mouse longevity data are also available


## Formal Mentorship ##

-   [2023] **Cindy Pino-Barrios**
    (Rotation Student in Biological and Medical Informatics, UCSF)
    - Served as a co-primary mentor during her 10-week rotation in the
      Pollard Lab, where she initiated a project studying the use of
      single-nucleotide polymorphisms in the core genome to predict the
      presence/absence of functional gene families in a publicly available
      reference genome collection. This project has become the basis for a
      manuscript that is in preparation.
    - Currently: Graduate Student working with Prof. Katie Pollard, UCSF

-   [2023] **Claire Dubin**
    (Rotation Student in Bioinformatics, UCSF)
    - Served as a co-primary mentor during her 10-week rotation in the
      Pollard Lab, where she initiated a project tracking strain-resolved
      transmission outcomes in a large meta-study of fecal microbiota
      transplantation experiments.
    - Currently: Graduate Student, UCSF

-   [2019] **Erin Gilbertson**
    (Rotation Student in Biological and Medical Informatics, UCSF)
    - Served as Erin's primary mentor during her 10-week
      rotation in the Pollard Lab, where she worked on two separate
      studies during that time: (1) the analysis of a longitudinal,
      16S rRNA amplicon data in a pilot study comparing anti-TNF and
      Vedolizumab treatments for ulcerative colitis patients, and (2)
      a simulation study benchmarking the performance of StrainFinder,
      software for the deconvolution of metagenotype data.
    - Currently: PhD Candidate working with Prof. Tony Capra, UCSF



## Teaching ##

-   [2020] Guest Lecturer,
    **BMI 206: Statistical Methods of Bioinformatics**, University of California, San Francisco
    -   Prepared and taught an interactive lesson titled
        [_A Brief Introduction to Bayesian Inference_](https://byronjsmith.com/bmi206_bayesian.pdf)
        to first year PhD students in the Biological and Medical
        Informatics Graduate Program
    -   Course Instructor: Dr. Katherine S. Pollard

-   [2015-2020] Instructor,
    **Software Carpentry Bootcamps**,
    -   Co-taught over a dozen workshops on
        computing skills for researchers: programming, the command-line
        version control, data management, etc.
    -   Students are postdocs, graduate students, faculty, etc.
    -   Workshops at Stanford University, Harvard Medical School, University of
        Pennsylvania, University of California, San Francisco, University of
        New Hampshire, University of Michigan, Wayne State University,
        Lawrence Berkeley National Laboratory, and Genentech
    -   Contributed to development of shared teaching materials,
        notably lessons on the tools Git and Make.

-   [2015, 2018] Graduate Student Instructor,
    **Introductory Biology Lab: The Human Microbiome**,
    University of Michigan
    -   Assisted during three semesters in designing and running laboratory
        activities and lectures for a novel, research based, introductory
        biology lab
    -   Course Instructors: Dr. Thomas M. Schmidt, Dr. Kristi MacCready,
        and Dr. Arvind Venkateraman

-   [2017] Teaching Assistant,
    **Data Intensive Biology Summer Research Institute** hosted by
    the University of California, Davis
    -   Assisted with week-long workshops on "Environmental Metagenomics" and
        "Non-Model Organism RNA-Seq"
    -   Organizer: Dr. C. Titus Brown

-   [2016] Instructor,
    **Bodega Marine Laboratory Bioinformatics Workshop**,
    hosted by the University of California, Davis
    -   Invited to teach a half-day lesson titled
        [_GNU Make for Reproducible Analysis Pipelines_](https://blog.byronjsmith.com/make-analysis.html)
    -   Workshop Organizer: Dr. C. Titus Brown

-   [2014] Teaching Assistant, **Software Carpentry Bootcamp**,
    University of Michigan
    -   Assisted instructors teaching basic computing skills to
        early career scientists

-   [2013] Graduate Student Instructor, **General Ecology**,
    University of Michigan
    -   Course Instructor: Dr. Aaron A. King

-   [2012] Teaching Assistant, **Microbial Genomics Lab**,
    Michigan State University
    -   Course Instructor: Dr. Robert A. Britton


## Service and Outreach ##

-   [2019-present] **Gladstone Postdoc Advisory Committee** organizer and member
    -  Advised Gladstone leadership and administration on postdoc issues;
       involved in planning and executing numerous
       activities and events
    -  Lead postdoc advocacy efforts to increase representation, recruitment,
       and retention of postdocs from communities historically
       under-represented in the life sciences, which directly contributed to
       the creation and implementation of the Gladstone DEI Strategic Plan.

-   [2012-present] **Peer Reviewer**
    - Reviewed submitted manuscripts at _eLife_, _Microbiome_, _Cell Systems_,
      _PLoS Computational Biology_, _PLoS Biology_, _Inflammatory Bowel Disease_,
      _Scientific Reports_, _ISME_, and others.

-   [2018-2023] **Software Carpentry Curriculum Advisory Committee**
    - [2022-2023] Committee Chair

-   [2019] **UCSF Library Reproducibility for Biomedical Researchers** panelist

-   [2016-2018] **University of Michigan Software Carpentry** group member
    -   Actively involved in developing a community of Software and Data
        Carpentry instructors, learners, and advocates

-   [2017] **Models: At the Intersection of Data and Discovery.**
    -   Collaborated in proposing a meeting for the directors
        and students of the Burroughs Wellcome Fund's Institutional Program
        Unifying Population and Laboratory Based Sciences
    -   Held at the University of Michigan
    -   \$150,000 budget

-   [2016-2017] **Ecology and Evolutionary Biology Statistics Office Hours**
    organizer
    -   Co-founded and organized monthly statistics help sessions for graduate
        students

-   [2016] Graduate Student Representative to the Department of Ecology
    and Evolutionary Biology **Executive Committee** during the fall term

-   [2014-2015] Graduate Student Representative to the
    Department of Ecology and Evolutionary Biology **Seminar Committee**

-   [2015] National Ocean Sciences Bowl, "**Great Lakes Bowl**" volunteer

-   [2013-2014] Michigan **Xplore Engineering**
    "Microbiology and Environmental Engineering" workshop team
